packages/strapi-plugin-graphql/services/Mutation.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
           : {};
 
         // Retrieve the controller's action to be executed.
-        const [name, action] = handler.split('.');
+        [name, action] = handler.split('.');
```

Comment:
Can you please avoid mutating the name and action variables.

Reply:
But in case of a custom mutation the correct 'name' and 'action' are not available until this point I think. And I need that 'name' and 'action' outside that statement. What's the best way to achieve that?

Reply:
I would split the code in smaller functions so you can just have `if (action) createActionMutation` else `createCustomMutation`

Reply:
At what place inside the Mutation.js do I need to create this? I don't have it clear for myself how you want to do it. Can you maybe give me an example or something?

Reply:
I basically just want you to do `const [controllerName, ctrollerAction] = handler.split('.')` :)

Reply:
I found another way that works. Will push in a minute. Let me know what you think :)